test/query_generator/fragment_generator_test.dart

```diff
@@ -0,0 +1,316 @@
+                          FragmentSpreadNode(
+                              name: NameNode(value: 'weight'), directives: [])
+                        ])),
+                    FieldNode(
+                        name: NameNode(value: 'attacks'),
+                        alias: null,
+                        arguments: [],
+                        directives: [],
+                        selectionSet: SelectionSetNode(selections: [
+                          FragmentSpreadNode(
+                              name: NameNode(value: 'pokemonAttack'),
+                              directives: [])
+                        ]))
+                  ])),
+              FragmentDefinitionNode(
+                  name: NameNode(value: 'weight'),
+                  typeCondition: TypeConditionNode(
+                      on: NamedTypeNode(
+                          name: NameNode(value: 'PokemonDimension'),
+                          isNonNull: false)),
+                  directives: [],
+                  selectionSet: SelectionSetNode(selections: [
+                    FieldNode(
+                        name: NameNode(value: 'minimum'),
+                        alias: null,
+                        arguments: [],
+                        directives: [],
+                        selectionSet: null)
+                  ])),
+              FragmentDefinitionNode(
+                  name: NameNode(value: 'pokemonAttack'),
+                  typeCondition: TypeConditionNode(
+                      on: NamedTypeNode(
+                          name: NameNode(value: 'PokemonAttack'),
+                          isNonNull: false)),
+                  directives: [],
+                  selectionSet: SelectionSetNode(selections: [
+                    FieldNode(
+                        name: NameNode(value: 'special'),
+                        alias: null,
+                        arguments: [],
+                        directives: [],
+                        selectionSet: SelectionSetNode(selections: [
+                          FragmentSpreadNode(
+                              name: NameNode(value: 'attack'), directives: [])
+                        ]))
+                  ])),
+              FragmentDefinitionNode(
+                  name: NameNode(value: 'attack'),
+                  typeCondition: TypeConditionNode(
+                      on: NamedTypeNode(
+                          name: NameNode(value: 'Attack'), isNonNull: false)),
+                  directives: [],
+                  selectionSet: SelectionSetNode(selections: [
+                    FieldNode(
+                        name: NameNode(value: 'name'),
+                        alias: null,
+                        arguments: [],
+                        directives: [],
+                        selectionSet: null)
+                  ]))
+            ]));
+      }, count: schemeMapping.length);
+
+      await testBuilder(anotherBuilder, {
```

Comment:
Can you test the generated Dart file for this builder [like some other tests](https://github.com/comigor/artemis/blob/master/test/query_generator/query_generator_test.dart#L106-L129)?